### ISSUE TITLE
api: get organization failed with 403

### ISSUE THREAD

**Issue by alimcodescalers** [itsyouonline/identityserver#233]
```
itsyouonline_testsuite: INFO: Testcase test001_get_organization Started at 1470152026.0
requests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): itsyou.online
requests.packages.urllib3.connectionpool: DEBUG: "POST /v1/oauth/access_token?client_secret=dZwkaFmPzyYAbdJaamO8bOtDnrDs&grant_type=client_credentials&client_id=3-RR68hSplZUWRoWN-lov9IKx4nU HTTP/1.1" 200 None
requests.packages.urllib3.connectionpool: DEBUG: "GET /api//users/alim/organizations HTTP/1.1" 301 0
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/users/alim/organizations HTTP/1.1" 200 47
itsyouonline_testsuite: INFO: GetUserOrganizations [alim] response [{u'member': [], u'owner': [u'codescalers', u'google']}]
itsyouonline_testsuite: INFO: organization_id codescalers
itsyouonline_testsuite: INFO: test001_get_organization STARTED
requests.packages.urllib3.connectionpool: DEBUG: "GET /api//organizations/codescalers HTTP/1.1" 301 0
requests.packages.urllib3.connectionpool: DEBUG: "GET /api/organizations/codescalers HTTP/1.1" 403 0
```

**Comment by alimcodescalers** [comment]
also get organization contracts, get organization invitations, and get organization apikeys are failing with 403

**Comment by lzonneke** [comment]
still available in 7f2eafc
